wrong required cmake version 3.18

```sh
CMake Error at upgrade_test/CMakeLists.txt:35 (cmake_path):
  Unknown CMake command "cmake_path".
```
command `cmake_path` is appeared in version 3.20 https://cmake.org/cmake/help/v3.20/command/cmake_path.html

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,5 +1,5 @@
-cmake_minimum_required(VERSION 3.18)
-# file(ARCHIVE_EXTRACT foo) need 3.18
+cmake_minimum_required(VERSION 3.20)
+# cmake_path requires 3.20
 
 project(diskquota)
 

